cmd/utils/flags.go

```diff
@@ -1866,6 +1872,11 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *ethconfig.Config) {
 	cfg.RollupDisableTxPoolAdmission = cfg.RollupSequencerHTTP != "" && !ctx.Bool(RollupEnableTxPoolAdmissionFlag.Name)
 	cfg.RollupHaltOnIncompatibleProtocolVersion = ctx.String(RollupHaltOnIncompatibleProtocolVersionFlag.Name)
 	cfg.ApplySuperchainUpgrades = ctx.Bool(RollupSuperchainUpgradesFlag.Name)
+
+	if ctx.IsSet(RollupSequencerEnableTxConditionalFlag.Name) {
+		cfg.RollupSequencerEnableTxConditional = ctx.Bool(RollupSequencerEnableTxConditionalFlag.Name)
+	}
```

Comment:
It's fine to just rely on the boolean false default, no need to check if it's set first. See above `cfg.ApplySuperchainUpgrades` usage.